fulp_modules/features/halloween/_code.dm

```diff
@@ -64,6 +64,9 @@
 /obj/item/storage/box/halloween/edition_21
 	year = 2021
 
+/obj/item/storage/box/halloween/edition_22
+	year = 2022
```

Comment:
oh sweet jesus i've never paid attention to this, what a horrible way to do this. dont bother changing it, if it works at the moment i'm not interested in changing it but i think that if i want to rehabilitate fulp code for ss13 the fulp_modules section will need some attention.

Reply:
the fulp_modules section is all fulp code

also if the year is used in the name, then it's a fine way to have it?